project/SparkBuild.scala

```diff
@@ -254,7 +254,7 @@ object SparkBuild extends PomBuild {
   val mimaProjects = allProjects.filterNot { x =>
     Seq(
       spark, hive, hiveThriftServer, catalyst, repl, networkCommon, networkShuffle, networkYarn,
-      unsafe, testTags, sketch, mllibLocal
+      unsafe, testTags, sketch, mllibLocal, compatibility
```

Comment:
Looks like we should move sketch and compatibility out from this list once we release 2.0, right?